af/oometa/014-ooexpl-insert.rollback.sql

```diff
@@ -2,6 +2,6 @@ BEGIN;
 
 select _v.unregister_patch( '014-ooexpl-insert');
 
-TRUNCATE ooexpl_recent_msm_count, ooexpl_daily_msm_count;
+TRUNCATE ooexpl_bucket_msm_count, ooexpl_daily_msm_count;
```

Comment:
BTW, should it be `TRUNCATE` or `DROP TABLE` ?

Reply:
In this case TRUNCATE is correct, because this item is just about INSERTING into the table. The `DROP` instruction goes in the 013 `rollback` script.

Reply:
Ah, true. Sorry, I was a bit confused.